components/static-pages/tos.md

```diff
@@ -44,7 +44,7 @@ This Section applies to you if you use the Platform as a Financial Contributor.
 
 (c) **Recurring Payments.** You may authorize a recurring contribution to a Host for designation to a Collective by indicating as much through the Platform ("**Recurring Contribution**"). For a Recurring Contribution, you are authorizing our designated third-party payment service to charge the amount specified to your Payment Method on the day that the initial contribution is made and to charge the same amount at the beginning of every calendar month thereafter (or the beginning of every year if the renewal period is annual). For example, if you initiate a recurring payment of $10 on January 25th, you will be charged $10 on that day, and \$10 on February 1st and the first day of every calendar month thereafter until you cancel. WHEN YOU INITIATE A RECURRING CONTRIBUTION TO A HOST FOR DESIGNATION TO A COLLECTIVE, YOU EXPRESSLY AGREE THAT OUR THIRD-PARTY PAYMENT SERVICE IS AUTHORIZED TO CHARGE YOU ON A MONTHLY BASIS (OR ANNUAL BASIS IF SPECIFIED VIA THE PLATFORM) FOR THE SAME PAYMENT AMOUNT UNTIL YOU CANCEL IT OR WE SUSPEND OR STOP PROVIDING YOU ACCESS TO THE PLATFORM IN ACCORDANCE WITH THESE TERMS.
 
-(d) **Cancellations.** Financial Contributors can cancel or change the amount of a Recurring Contribution via [opencollective.com/subscriptions](http://www.opencollective.com/subscriptions).
+(d) **Cancellations.** Financial Contributors can cancel or change the amount of a Recurring Contribution via [opencollective.com/recurring-contributions](http://www.opencollective.com/recurring-contributions).
```

Comment:
If possible, our canonical address should be in the `https://opencollective.com/` form (https, no www).